modules/builds-strategy-docker-build-arguments.adoc

```diff
@@ -6,6 +6,11 @@
 
 You can set link:http://docs.docker.com/v1.7/reference/api/hub_registry_spec/#docker-registry-1-0[docker build arguments] using the `BuildArgs` array. The build arguments are passed to docker when a build is started.
 
+[TIP]
+====
+Consider reviewing link:https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact[Understand how ARG and FROM interact] in the Dockerfile reference documentation.
```

Comment:
'Consider reviewing' seems to be similar to using 'please'. How about simply saying saying: Review
Also '...reviewing Understand how ARG and FROM interact...' seems a bit awkward, how about:  'reviewing  the Understand how ARG and FROM interact section ...